onnx_tf/common/handler_helper.py

```diff
@@ -23,19 +23,19 @@ def get_all_frontend_handlers(opset_dict):
     version = opset_dict[domain]
     handler.VERSION = version
 
-    since_version = 1
     if handler.ONNX_OP and defs.has(handler.ONNX_OP, domain=handler.DOMAIN):
-      since_version = defs.get_schema(
+      handler.SINCE_VERSION = defs.get_schema(
```

Comment:
@tjingrant The since_version is always 1 if the frontend tf op has no matching onnx op to map to. This assumption is incorrect because we often add new frontend tf op support in later versions and we don't want the handler to be tagged as SINCE_VERSION = 1.